virocon/distributions.py

```diff
@@ -411,12 +673,46 @@ def _fit_lsq(self, data):
         
 class ExponentiatedWeibullDistribution(Distribution):
     """
-    An exponentiated Weibull distribution.
+    An exponentiated Weibull distribution by by Haselsteiner et al. (2019) [1]_
+    
+    :math:`F(x) = \\left[ 1- \\exp \\left(-\\left( \\frac{x}{\\alpha} \\right)^{beta} \\right) \\right] ^{\\delta}`
+    
+    Parameters
+    ----------
+    alpha : float
+        Scale parameter of the exponentiated weibull distribution. Defaults 
+        to 1.
+    beta : float
+        Shape parameter of the exponentiated weibull distribution. Defaults 
+        to 1.
+    delta : float
+        Shape 2 parameter of the exponentiated weibull distribution. Defaults 
+        to 1.
+    f_alpha : float
+        Fixed scale parameter of the weibull distribution (e.g. given physical
+        parameter). If this parameter is set, alpha is ignored. Defaults to 
+        None.
+    f_beta : float
+       Fixed shape parameter of the weibull distribution (e.g. given physical
+       parameter). If this parameter is set, beta is ignored. Defaults to 
+       None. 
+    f_delta : float
+        Fixed location parameter of the weibull distribution (e.g. given physical
+        parameter). If this parameter is set, delta is ignored. Defaults to 
+        None.
+    fit_method : float
+        Method of estimating the parameters of a probability distribution. 
+        Defaults to maximum likelihood estimation (mle).
+    weights : KAI???
```

Comment:
array_like, float or str
The weights for the weighted least squares fit, only used when fit_method="lsq", ignored otherwise. Defaults to None i.e. equal weights. TODO: there might still be a bug here...